graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/ScalarsDmlTest.java

```diff
@@ -185,28 +186,20 @@ public void incorrectLiteralsForScalarsShouldResultInError(Column.Type type, Obj
         arguments(Column.Type.Int, 0, null),
         arguments(Column.Type.Int, Integer.MAX_VALUE, null),
         arguments(Column.Type.Int, Integer.MIN_VALUE, null),
-        arguments(Column.Type.Smallint, (short) 0, null),
-        arguments(Column.Type.Smallint, (short) -1, null),
-        arguments(Column.Type.Smallint, (short) 1, null),
-        arguments(Column.Type.Smallint, Short.MAX_VALUE, null),
-        arguments(Column.Type.Smallint, Short.MIN_VALUE, null),
+        arguments(Column.Type.Smallint, 0L, null),
```

Comment:
SmallInt and TinyInt are serialized as longs now. Apart from this test, it makes no difference because GraphQL and JSON don't differentiate integer types.